templates/status_email@.service.erb

```diff
@@ -0,0 +1,8 @@
+[Unit]
```

Comment:
can you convert this into an epp template and maybe add a comment like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
done